components/rate/rate.component.ts

```diff
@@ -221,6 +221,7 @@ export class NzRateComponent implements OnInit, ControlValueAccessor, OnChanges
 
   onItemHover(index: number, isHalf: boolean): void {
     if (this.nzDisabled || (this.hoverValue === index + 1 && isHalf === this.hasHalf)) {
+      this.nzOnHoverChange.emit(this.hoverValue);
```

Comment:
If the state is disabled, the event should not be triggered.

Reply:
done